38/quickstart.html

```diff
@@ -237,7 +237,7 @@ <h4 class="anchor-heading">
             Edit the <code class="language-bash">config/connect-standalone.properties</code> file, add or change the <code>plugin.path</code> configuration property match the following, and save the file:
         </p>
 
-        <pre class="line-numbers"><code class="language-bash">$ echo "plugin.path=libs/connect-file-{{fullDotVersion}}.jar &gt;&gt; config/connect-standalone.properties"</code></pre>
+        <pre class="line-numbers"><code class="language-bash">$ echo "plugin.path=libs/connect-file-{{fullDotVersion}}.jar" &gt;&gt; "config/connect-standalone.properties"</code></pre>
```

Comment:
`"config/connect-standalone.properties"` -> `config/connect-standalone.properties`

just for consistency to https://github.com/apache/kafka/pull/16200

Reply:
Ah, good catch! Thanks, done.